framework/audio/iaudioengine.h

```diff
@@ -51,6 +51,7 @@ class IAudioEngine : MODULE_EXPORT_INTERFACE
     virtual Ret init() = 0;
     virtual void deinit() = 0;
     virtual bool isInited() const = 0;
+    virtual async::Channel<bool> initChanged() const = 0;
```

Comment:
Can you explain to me please, what this notification is needed for?

Reply:
To initialize the synthesizer, we need a sample rate. The sample rate can be obtained only after initializing the audio engine. So we can initialize the synthesizer only after initializing the audio engine. If the audio engine is not initialized at the moment, I need to subscribe and wait for it to be initialized.
https://github.com/musescore/MuseScore/pull/6399/files#diff-57c3ba1eac6f3d5a4de940828b780ad2R38